zingolib/src/blaze/block_witness_data.rs

```diff
@@ -827,6 +817,20 @@ impl BlockAndWitnessData {
     }
 }
 
+fn is_orchard_tree_verified(determined_orchard_tree: String, unverified_tree: TreeState) -> bool {
```

Comment:
I really like this.   I think this helperization/test/bug reproduction belongs in its own PR.  Should we split it out?

Reply:
good Idea. I'll split it out and send a new PR with that change.

Reply:
@zancas  I created a PR for this! #298